src/App.js

```diff
@@ -5,17 +5,33 @@ import data from './data/dummy.json';
 const App = () => {
   return (
     <div>
-      <ol>
-        {data.map((question) => {
-          return <div>
-            <li key={question.id}>{question.text}</li>
-            <ul>{question.answers.map((answer) => {
-              return <li key={answer.id}>{answer.text}</li>
-            })}
-            </ul>
-          </div>
-        })}
-      </ol>
+      <form>
+        <ol>
+          {data.map(question => {
+            return (
+              <div>
+                <li key={question.id}>{question.text}</li>
+
+                {question.answers.map(answer => {
+                  return (
+                    <div>
+                      <input
+                        type="radio"
+                        id={`${question.id}.${answer.id}`}
```

Comment:
Good job with creating an `id` using both `question` and `answer` `id`s 👍

Reply:
it was @AightA  idea  :)